### PR TITLE
[16.0][MIG] stock_picking_product_barcode_report: Migration to 16.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT47636

ping @sergio-teruel @carlosdauden

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]

File: stock_picking_product_barcode_report/report/report_label_barcode_template.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
                                     t-value="'02' + barcode + '10' + lot.name"
                                 />
                                 <img
-                                    t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' % (type, value, 600, 100)"
+                                    t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' % (type, value, 600, 60)"
```
Comment (pedrobaeza) [inline stock_picking_product_barcode_report/report/report_label_barcode_template.xml:21]: Why this change? Is doing it a bit shorter in height?

Comment (CarlosRoca13) [reply stock_picking_product_barcode_report/report/report_label_barcode_template.xml:21]: This prevents a blank space from appearing below the barcode. What really determines the size is the element's style.

Comment (pedrobaeza) [reply stock_picking_product_barcode_report/report/report_label_barcode_template.xml:21]: Then I understand this should be backported to v15 as well.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump
